mathics/builtin/compile/__init__.py

```diff
@@ -14,3 +27,96 @@
     from .compile import _compile
     from .base import CompileArg, CompileError
     from .types import *
+
+
+# _Complex not implemented
+permitted_types = {
+    Expression(SymbolBlank, SymbolInteger): int_type,
+    Expression(SymbolBlank, SymbolReal): real_type,
+    SymbolTrue: bool_type,
+    SymbolFalse: bool_type,
+}
+
+
+class CompileDuplicateArgName(Exception):
```

Comment:
Please let's not put clases in __init__.py

Reply:
OK. I left this here because I was not sure where is the best place to put it: some options I had in mind are

* mathics.builtin.compile.topython
* mathics.core.convert.python
* mathics.core.evaluators
* mathics.core.expression
* Another option?

Reply:
Smaller modules would be appreciated. So given though choices above, a new module in `mathics.builtin.compile` is preferable.